contents.json

```diff
@@ -463,7 +463,7 @@
   }, {
     "title": "Phone Numbers",
     "id": "phone-numbers",
-    "parent": "validation"
+    "parent": "Libs to manage phone numbers."
```

Comment:
Hi @NikKovIos, this is not a description, this is an ID. Could you revert this change and move this change to `title`?

Reply:
Done.